src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/IndexProperties.sql

```diff
@@ -0,0 +1,14 @@
+﻿GO
+CREATE TABLE dbo.IndexProperties 
+  (
+     TableN     varchar(100)     NOT NULL
```

Comment:
>TableN [](http://example.com/codeflow?start=5&length=6)

TableName

Reply:
Sorry, I will modify all the incorrect names.

Reply:
Please change to TableName

Reply:
I have tried to change name to TableName, but it will conflict with the TableName property of table, so we change the name to IndexTableName

Reply:
Original script has TableName column and it works. Please explain which property you refer to.